### ISSUE TITLE
How to pass configuration without using a JSON/PEM file?

### ISSUE THREAD

**Issue by furkee** [googleapis/nodejs-recaptcha-enterprise#269]
To assess the token generated via recaptcha enterprise: I created a service account on the cloud, created & downloaded keys, and set `GOOGLE_APPLICATION_CREDENTIALS` environment variable to point to the downloaded JSON file and I was able to make the assessment request. However, for the secret storing setup my application has, this is not a viable solution. I could not figure out a way to pass the necessary parameters to `RecaptchaEnterpriseServiceClient` without using `keyFile` argument. Is there way to achieve this?

TL:DR; I want to pass client_id, private_key etc without using `keyFile`. Can I pass these options to constructor? If so, how? Neither constructor nor the samples make it clear which variables are actually needed to make a successful assessment request.

**Comment by furkee** [comment]
Update: I tried removing to find out which fields are actually required from the JSON file by removing them 1 by 1. It turns out the recaptcha client only needs `private_key` and `client_email` fields from the JSON file that is generated after created API keys for a service account. It is possible to pass this values to the recaptcha client under `credentials` property in the constructor as in:

```js
new RecaptchaEnterpriseServiceClient({ credentials: { client_email: '..', private_key: '..' }});
```
